bindings/python/crocoddyl/core/solvers/fddp.cpp

```diff
@@ -46,23 +46,6 @@ void exposeSolverFDDP() {
                ":param regInit: initial guess for the regularization value. Very low values are typical\n"
                "                used with very good guess points (init_xs, init_us) (default None).\n"
                ":returns the optimal trajectory xopt, uopt and a boolean that describes if convergence was reached."))
-      .def("computeDirection", &SolverFDDP::computeDirection,
-           SolverFDDP_computeDirections(
-               bp::args("self", "recalc"),
-               "Compute the search direction (dx, du) for the current guess (xs, us).\n\n"
-               "You must call setCandidate first in order to define the current\n"
-               "guess. A current guess defines a state and control trajectory\n"
-               "(xs, us) of T+1 and T elements, respectively.\n"
-               ":params recalc: true for recalculating the derivatives at current state and control.\n"
-               ":returns the search direction dx, du and the dual lambdas as lists of T+1, T and T+1 lengths."))
-      .def("expectedImprovement", &SolverFDDP::expectedImprovement,
```

Comment:
Why are you removing `expectedImprovement`?

Reply:
This is a duplicated code. Note that this is a virtual function which it reimplemented in this class. Furthermore, this function is already bind. Therefore, we don't need to rebind it again.

The historical context is I have created this code because it was not properly defined as virtual in the first implementation.

Reply:
great, thanks.